common/shared.c

```diff
@@ -209,7 +210,7 @@ static uint64_t ft_caps_to_mr_access(uint64_t caps)
 {
 	uint64_t mr_access = 0;
 
-	if (caps & FI_MSG) {
+	if ((caps & FI_MSG) || (caps & FI_TAGGED)) {
```

Comment:
nit: caps & (FI_MSG | FI_TAGGED)   instead?

but, yes, this looks like this was missed.